src/test/ultimate.test.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { render, shallow, mount } from 'enzyme';
+import * as _ from 'lodash';
+
+import * as ultimate from '../pages/ultimate';
+
+describe('ultimate tic-tac-toe', () => {
+  it('Nought matches the snaphot', () => {
+    expect(render(<ultimate.Nought />)).toMatchSnapshot();
+  });
+
+  it('Cross matches the snaphot', () => {
+    expect(render(<ultimate.Cross />)).toMatchSnapshot();
+  });
+
+  it('Square renders nothing with no winner', () => {
+    const component = render(
+      <ultimate.Square cell={{ player: null }} onClick={() => null} />,
+    ).get(0);
+    expect(component.children).toEqual([]);
+  });
+
+  it('Square renders winner', () => {
+    const component = render(
+      <ultimate.Square
+        cell={{ player: ultimate.Winner.Cross }}
+        onClick={() => null}
+      />,
+    );
+    expect(component.find('svg').length).toBeGreaterThanOrEqual(1);

```

Comment:
`expect(component).toContainSelector('svg')` not an option? (Now that the tests pass.)